src/pages/admin/company-profile/components/company-profile-logo-header/CompanyProfileLogoHeader.module.scss

```
@@ -0,0 +1,122 @@
+
+.company-logo-icon {
+    width: 100%;
+    height: auto;
+    max-height: 100%;
+    fill: c.$blue-800;
+}
+
+.company-title {
+    margin: 0;
+    @include type.h1-black;
+    color: c.$blue-800;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)